<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network</class>
 <widget class="QMainWindow" name="Network">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="machine1Widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>70</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_mac01_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mac01_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="machine2Widget" native="true">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>10</y>
      <width>131</width>
      <height>70</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_mac02_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mac02_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="networkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>10</y>
      <width>231</width>
      <height>70</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit_net_ip">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_net_port_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_net_port_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="errorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>331</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_error">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>65</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Error Rate</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_BER">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_error">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Delay">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay(ms)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Delay">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>108</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
